packages/docs/pages/users/wallet/hardware-wallet.mdx

```diff
@@ -40,14 +40,17 @@ Scroll down until you find Namada in the list, and select the newest version fro
 
 </Steps>
 
-## Using the Ledger app with the browser extension
+## Using a Ledger with Namada Keychain (browser extension)
+
```

Comment:
I think we need a disclaimer here that this will only work on Chrome-enabled browsers. Firefox removed the USB transport. We only show `Connect to Hardware Wallet` when it is available in the browser.